README.md

```diff
@@ -305,4 +308,5 @@ are required. After that the configuration will be removed.
 [node]: https://nodejs.org/
 [ns]:   https://nodesource.com/
 [pivotal]: https://www.pivotalagency.com.au/
+[release secrets]: https://github.com/nodejs-private/secrets/tree/master/build/release/.gpg
```

Comment:
remove `.gpg` from the end of that

Reply:
Its intentional - the .gpg directory contains the list of people with access